src/event/sys/windows/parse.rs

```diff
@@ -79,17 +79,21 @@ fn parse_key_event_record(key_event: &KeyEventRecord) -> Option<KeyEvent> {
             let character_raw = key_event.u_char;
 
             if character_raw < 255 {
```

Comment:
```suggestion
            if character_raw < 255 && character_raw != 0 {
```

Reply:
This is not equivalent. with this change if character_raw ==0 it will still return Some(Keycode)